### ISSUE TITLE
omv6 type context menu button does not get disabled with enabledConstraints

### ISSUE THREAD

**Issue by subzero79** [openmediavault/openmediavault#1015]
**Is your feature request related to a problem? Please describe.**
Trying to create a context a menu for porting the luks encryption plugin. We have two context buttons that expand so i know we have more space now but i want to know if is posible to implement or it needs to be declared in another way

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/7058119/115956741-e0ecba80-a541-11eb-9628-b03486d2487a.png)

above omv5, now with omv6 using this solution:

```yaml
      - type: menu
        icon: "mdi:key"
        tooltip: _("Manage Keys")
        enabledConstraints:
          minSelected: 1
          maxSelected: 1
        actions:
          - type: iconButton
            # tooltip: _("Add")
            icon: "mdi:plus"
            text: _("Add")
            execute:
              type: url
              url: "/storage/encryption/addkey"
```

![image](https://user-images.githubusercontent.com/7058119/115956782-2d37fa80-a542-11eb-867a-c3daa18fa58b.png)

Menu is enabled but no device selected

**Describe alternatives you've considered**

**Additional context**

Is it possible to auto generate the docs front end plugin development? 

Also there is not hot reload to work on this, so i am editing directly the yaml files over the installed path. as a suggestion there should be an aditional path like /usr/share/openmediavault/workbench-dev to work over the files while in development so you don't get confused with existing files or naturally installed plugins. Unless there is another workflow I am not aware

![image](https://user-images.githubusercontent.com/7058119/115956921-e696d000-a542-11eb-9bc1-443eea70ea88.png)

**Comment by subzero79** [comment]
It would also be great if the omv-mkworkbench implements or leaves sort of trail that it ran, or marks a file dirty so the backend can pick it up and somehow signal the frontend to trigger a reload. With that and cache disabled frontend dev would be easier IMO. Doesn't have to be enabled by default it would be only for dev purposes

**Comment by votdev** [comment]
Fixed with c60d1a09e9353a088c8bf8dd534745f605a450b3.

**Comment by votdev** [comment]
> It would also be great if the omv-mkworkbench implements or leaves sort of trail that it ran, or marks a file dirty so the backend can pick it up and somehow signal the frontend to trigger a reload. With that and cache disabled in browser the frontend develp experience would be easier IMO.
> Doesn't have to be enabled by default it would be only for dev purposes

Fixed by a730089b55e81806690b01780a695d47c7fd8649.
